echo-model/src/main/java/com/netflix/spinnaker/echo/model/Trigger.java

```diff
@@ -65,6 +65,10 @@ public String toString() {
   String type;
   boolean enabled;
 
+  // Configuration for pipeline triggers
+  String parentPipelineId;
+  String paraentPipelineApplication;
```

Comment:
`parentPipelineApplication`, also I don't see it use anywhere other than a few tests. so probably not needed

Reply:
it looked like orca only needed parentPipelineId however i did notice that parentPipelineApplication was passed by deck as part of the trigger so i included it for completeness in case the field is needed in the future.

Reply:
ill fix the typo